delta/plugins/storage/src/main/resources/contexts/storages.json

```diff
@@ -5,6 +5,12 @@
     "_storageId": {
       "@id": "https://bluebrain.github.io/nexus/vocabulary/storageId",
       "@type": "@id"
+    },
+    "files": "https://bluebrain.github.io/nexus/vocabulary/files",
```

Comment:
`files` and `spaceUsed` are not needed here, since you have the `@vocab`.
As you can see, the ony things defined here are the ones that have a custom `@type`